src/EventSubscriber/GitLabWebhookSubscriber.php

```diff
@@ -84,10 +84,8 @@ public function onController(ControllerEvent $event): void {
 			// Resolve events
 			$webhookEvents = [];
 			foreach ($annotations as $annotation) {
-				$webhookEvents[$event = $annotation->getEvent()] ??= [];
-				if (null !== ($token = $annotation->getToken())) {
-					$webhookEvents[$event][] = $token;
-				}
+				$tokens = $webhookEvents[$event = $annotation->getEvent()] ??= [];
```

Comment:
```suggestion
				$tokens = ($webhookEvents[$event = $annotation->getEvent()] ??= []);
```

This will work as is, but we could add brackets just for clarity's sake because an assignment is being done within an assignment.